### ISSUE TITLE
"Do Instructors pay for travel?" - the answer here says the host must budget "about $2000 total ($1000 per Instructor) for the travel, food, and accommodation of the Instructors". But under the "Travel Expenses for Instructors" on the Host a workshop page, we state that "we recommend estimating $3,000 total for expenses ($1500 per Instructor)". Can we have one price, please @sheraaronhurt or @danielle06.

### ISSUE THREAD

**Issue by maneesha** [carpentries/new-website#231]

**Comment by maneesha** [comment]
Question raised by @OscarSiba 

> "Do Instructors pay for travel?" - the answer on workshops FAQ says the host must budget "about $2000 total ($1000 per Instructor) for the travel, food, and accommodation of the Instructors". But under the "Travel Expenses for Instructors" on the [Host a workshop page](https://carp-new-website.netlify.app/workshops/host-workshop/#travel-expenses-for-instructors), we state that "we recommend estimating $3,000 total for expenses ($1500 per Instructor)". Can we have one price, please @sheraaronhurt or @danielle06.

**Comment by sheraaronhurt** [comment]
@OscarSiba yes, the total should be $3000 ($1500/ per Instructor). Note this has increased to align with the cost of living increases.
